src/components/text-input/TextInput.scss

```diff
@@ -1,55 +1,105 @@
 @import '../../styles/variables';
+@import '../../styles/inputs';
 
-/**************************************
- * Text Input
- **************************************/
+.bdl-TextInput-input {
+    @include common-typography;
```

Comment:
For similar reasons of not being nested inside a container, and also usefulness inside a tethered component